tests/unit/test_observer.py

```diff
@@ -43,14 +56,20 @@ def test_filter_finite() -> None:
     ([3], [3, 1]),  # query_points missing a dimension
     ([3, 4, 2], [3, 1]),  # query_points have too many dimensions
 ])
-def test_filter_finite_raises_for_invalid_shapes(qp_shape: List[int], obs_shape: List[int]) -> None:
+def test_filter_finite_raises_for_invalid_shapes(qp_shape: ShapeLike, obs_shape: ShapeLike) -> None:
     with pytest.raises(ValueError):
         filter_finite(tf.ones(qp_shape), tf.ones(obs_shape))
 
 
 def test_map_is_finite() -> None:
-    query_points = tf.constant([[0., 0.]] + [[-1., 0.], [1., 0.], [0., 2.], [1., 3.]])
-    is_finite = map_is_finite(query_points, _nan_at_origin(query_points))
+    query_points = tf.constant([[-1., 0.], [1., 0.], [0., 2.], [0., 0.], [1., 3.]])
+    is_finite = map_is_finite(query_points, _sum_with_nan_at_origin(query_points))
```

Comment:
Same comment as above regarding extracting a named variable for the observations (same for the test below too).

Reply:
will do